### PR TITLE
Solidity bump to 8.20

### PR DESCRIPTION
**Pull request by emozilla**

### CODE REVIEW

**Review by jensendarren — APPROVED** [review]

File: contracts/FetchPlayground.sol [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.3;
```
Comment (KolomboPulse) [inline contracts/FetchPlayground.sol:2]: 3

File: contracts/FetchPlayground.sol [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+pragma solidity 0.8.20;
```
Comment (KolomboPulse) [inline contracts/FetchPlayground.sol:2]: 20

File: hardhat.config.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ require("@nomiclabs/hardhat-ethers");
 task("balance", "Prints an account's balance").setAction(async () => {});
 
 module.exports = {
-  solidity: "0.8.3",
```
Comment (KolomboPulse) [inline hardhat.config.js:10]: 3

File: hardhat.config.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ require("@nomiclabs/hardhat-ethers");
...
+  solidity: "0.8.20",
```
Comment (KolomboPulse) [inline hardhat.config.js:10]: 20

File: contracts/interface/IERC20.sol [hunk 5]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.3;
```
Comment (KolomboPulse) [inline contracts/interface/IERC20.sol:2]: 3

File: contracts/interface/IERC20.sol [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
+pragma solidity 0.8.20;
```
Comment (KolomboPulse) [inline contracts/interface/IERC20.sol:2]: 20

File: contracts/UsingFetchUpgradeReady.sol [hunk 7]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.3;
```
Comment (KolomboPulse) [inline contracts/UsingFetchUpgradeReady.sol:2]: 3

File: contracts/UsingFetchUpgradeReady.sol [hunk 8]
```diff
@@ -1,5 +1,5 @@
...
+pragma solidity 0.8.20;
```
Comment (KolomboPulse) [inline contracts/UsingFetchUpgradeReady.sol:2]: 20

File: contracts/UsingFetch.sol [hunk 9]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.3;
```
Comment (KolomboPulse) [inline contracts/UsingFetch.sol:2]: 3

File: contracts/UsingFetch.sol [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
+pragma solidity 0.8.20;
```
Comment (KolomboPulse) [inline contracts/UsingFetch.sol:2]: 20

File: contracts/interface/IERC20.sol (hunk 5 above)
Comment (KolomboPulse) [inline contracts/interface/IERC20.sol:2]: 3

File: contracts/interface/IERC20.sol (hunk 6 above)
Comment (KolomboPulse) [inline contracts/interface/IERC20.sol:2]: 20

File: hardhat.config.js (hunk 3 above)
Comment (KolomboPulse) [inline hardhat.config.js:10]: 3

File: hardhat.config.js (hunk 4 above)
Comment (KolomboPulse) [inline hardhat.config.js:10]: 20

File: contracts/FetchPlayground.sol (hunk 1 above)
Comment (KolomboPulse) [inline contracts/FetchPlayground.sol:2]: 3

File: contracts/FetchPlayground.sol (hunk 2 above)
Comment (KolomboPulse) [inline contracts/FetchPlayground.sol:2]: 20

**Review by KolomboPulse — COMMENTED** [review]
Solidity bump to 8.20 #3

**Pull request merged**